### PR TITLE
Milestone: Mobile menu

### PR DESCRIPTION
**Pull request by olaishola05**
# Add mobile menu navigation to the mobile version

- Add Dynamic mobile navigation
- Setup configs for linters
- Add app.js for javascript 
- Add nav links for mobile in the index.html file

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -60,10 +68,53 @@ li {
...
+  padding: 10px;
+  background: #6070ff;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  border: none;
+  margin-top: 50px;
+  font-weight: bold;
+}
+
```
Comment (descholar-ceo) [inline style.css]: - [ ] The mobile menu should be full screen in all screens starting from the smallest screen (320px of width), and also when we scroll down the mobile menu shouldn't be hidden on scroll down, it should stick in the screen until we click or tap one menu, kindly watch the following screencast for more context

https://user-images.githubusercontent.com/56886064/145872933-df1cd6bd-7221-4fef-b1aa-b44cbe849b90.mp4

**Review by descholar-ceo — CHANGES_REQUESTED** [review]
Hi @netman5 , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css [hunk 2]
```diff
@@ -60,10 +68,54 @@ li {
...
 }
 
-.navbar ul {
+.desktop-nav {
   display: none;
 }
 
+.mobile-menu {
+  position: absolute;
+  height: 750px;
```
Comment (alonsofalconi) [inline style.css]: - [ ] As the previous reviewer mentioned your mobile menu is sticking to the top when scrolling down so it is possible to see the content below. I think you could use `position: fixed` and `height: 100vh` so your mobile menu stays on top of the content and occupies 100% of the `viewport height` :wink:

image

**Review by alonsofalconi — CHANGES_REQUESTED** [review]
Hi @netman5 and @Anny85-code,

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.



 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css (hunk 2 above)
Comment (olaishola05) [reply style.css]: Thanks for the suggestion, effect the required changes

File: style.css [hunk 3]
```diff
@@ -60,10 +68,53 @@ li {
...
   display: none;
 }
 
+.mobile-menu {
+  position: fixed;
+  height: 100vh;
+  width: 100%;
+  top: 0;
+  visibility: hidden;
+  background: #6070ff;
```
Comment (carlos-ssh) [inline style.css:81]: # 💾 💾 💽 [OPTIONAL]   💽 💾 💾
#
- Provide the requested design in your background menu.

**Review by carlos-ssh — APPROVED** [review]
# 🍾🍾 **STATUS APPROVED** 👏🏽👏🏽

Hi @netman5 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Approved


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css (hunk 3 above)
Comment (olaishola05) [reply style.css:81]: Thanks, @carlos-ssh pls can you be more specific about the background menu design, I don't seem to understand what you requested me to do.

**Pull request merged**